api4/plugin.go

```diff
@@ -79,6 +80,15 @@ func uploadPlugin(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//Store bundle in file store
```

Comment:
nit: can you add a leading space to this comment? Also, I'm wondering if we want to add a bit of flavour here for a future reader:
```suggestion
	// Store bundle in the file store to allow access from other servers.
```

Reply:
✅